integrations/github/webhooks/webhook.go

```diff
@@ -97,10 +95,22 @@ func (h handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 				http.Error(w, "Bad Request", http.StatusBadRequest)
 				return
 			}
+
+			if payload != nil {
+				userCID = cid
+				break // Successful validation with non-empty payload - no need to repeat.
+			}
+		}
+
+		if payload == nil {
```

Comment:
Please comment that you couldn't find a connection for the event and that github requires 200 on successful delivery.

Reply:
done